package/python-kiwi-spec-template

```diff
@@ -530,6 +530,7 @@ BuildRequires:  dracut
 %endif
 Requires:       dracut-kiwi-lib = %{version}-%{release}
 Requires:       util-linux
+Requires:       util-linux-systemd
```

Comment:
This is a SUSE-only package, please guard it with `%if 0%{?suse_version}`.

Reply:
Done